### PR TITLE
Support net4.6 and separates serializers

### LINKED ISSUE #13: Separate Serialization into its own package
Motivation
====

Right now, serialization from `System.Text.Json` is only used for tests. People probably don't need it in everyday usage. It also causes issues for Mono users.

Note
---
This change should bump minor, version. Due to API incompatiblity.

### PR DESCRIPTION
**Pull request by Ygg01**
While dealing with #15 the #13 bit me in the backside, so I did two in one. 
@mtkennerly is this what you wanted? 

Your PR was ok, but there are some edge cases caused by #8 and your requirement.

fix #15 
fix #13

### CODE REVIEW

File: Linguini.Bundle.Test/Linguini.Bundle.Test.csproj [hunk 1]
```diff
@@ -24,4 +24,9 @@
         <AdditionalFiles Include="..\PluralRules.Generator\cldr_data\plurals.xml" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' != 'net5.0'">
```
Comment (mtkennerly) [inline Linguini.Bundle.Test/Linguini.Bundle.Test.csproj:27]: Some of these use `'$(TargetFramework)' != 'net5.0'`, and some use `'$(TargetFramework)' == 'net461'`. Was that intentional?

File: Linguini.Syntax.Tests/Linguini.Syntax.Tests.csproj [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
 
     <PropertyGroup>
-
         <IsPackable>false</IsPackable>
-
         <Nullable>enable</Nullable>
-
-        <TargetFrameworks>net5.0;netstandard2.1</TargetFrameworks>
-
+        <TargetFrameworks>net4.6.1;net5.0;netstandard2.1</TargetFrameworks>
```
Comment (mtkennerly) [inline Linguini.Syntax.Tests/Linguini.Syntax.Tests.csproj:6]: The other files use `net461`, and just this one uses `net4.6.1`. Probably doesn't matter, but just pointing it out in case.

**Review by mtkennerly — COMMENTED** [review]
> is this what you wantend?

Yes! Thank you very much :D

**Comment by Ygg01** [comment]
Ok. I will create a changelog, make sure tests run for 5.0 and 4.6.1 and throw away Mono tests, they were useless anyway.
